drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -172,6 +172,13 @@ class DragAndDropBlock(XBlock, XBlockWithSettingsMixin, ThemableXBlockMixin):
     block_settings_key = 'drag-and-drop-v2'
     has_score = True
 
+    def max_score(self):  # pylint: disable=no-self-use
+        """
+        Return the problem's max score, which for DnDv2 always equals 1.
```

Comment:
@nasthagiri, let me look into this one. I know we emit a different `max_value` [in our grade events](https://github.com/open-craft/xblock-drag-and-drop-v2/blob/3ad5a184dfe6ecb6644603cc75973e19738a97da/drag_and_drop_v2/drag_and_drop_v2.py#L697), so I'm not sure why we're returning a static value for the `max_score` method. There may be some legacy considerations here.